test/system/280-update.bats

```diff
@@ -161,4 +161,552 @@ device-write-iops   = /dev/zero:4000 | - | -
     run_podman rm -f -t0 testctr
 }
 
+function _create_container_check_change_of_HealthCheck_configuration_with_podman_update(){
+    local ctrname="$1"
+    local msg="$2"
```

Comment:
There is so much duplication here that it will become a maintenance nightmare. First and most obvious, `ctrname` and `msg` are never used by the caller. There is no reason to pass them as parameters. They should be defined in here.